pyronear/datasets/wildfire/frame_extractor.py

```diff
@@ -207,7 +207,7 @@ def _write_frames(self, labels: pd.DataFrame, path_to_frames: Union[str, Path])
             # Get the video
             movie = cv2.VideoCapture((self.path_to_videos / name).as_posix())
             # For each state
-            for index, row in group.iterrows():
+            for _, row in group.iterrows():
```

Comment:
Here, It depends what we want:
- Using conventional unindentified `_` to tell the developer **that this variable won't be used**
- Using explicit name `index` to tell the developer **how this variable could be used**

Look good to me either way.

Reply:
Agreed, I dug into the different options here and switched to `.itertuples`, which has the same behavior (without yielding indices) and expected to have a faster execution.

One could also use  `apply` syntax, I tried and it works but I feel like we would lose some code clarity in doing this.

Reply:
Indeed, `itertuples` is a better option!